### PR TITLE
feat: added ch2 and comments

### LINKED ISSUE #6: Comments
Document comments usage in Cairo

### PR DESCRIPTION
**Pull request by hurrikaanig**
Closes #6

### CODE REVIEW

File: src/ch02-04-comments.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Comments
+
+Text can be inserted into the body of Cairo programs to document notes about the code. The commented text is annotated with the // syntax. Text following it is ignored by the compiler.
```
Comment (enitrat) [inline src/ch02-04-comments.md:3]: Suggestion: "In Cairo programs, you can include explanatory text within the code using comments. To create a comment, use the `//` syntax, after which any text on the same line will be ignored by the compiler."

**Review by enitrat — APPROVED** [review]
lgtm, I added a rephrasing suggestion

File: src/ch02-04-comments.md (hunk 1 above)
Comment (fricoben) [reply src/ch02-04-comments.md:3]: + 1 👍

**Review by fricoben — APPROVED** [review]
guud

**Pull request merged**